src/vs/platform/extensionManagement/node/positronBootstrapExtensionsInitializer.ts

```diff
@@ -149,6 +151,31 @@ export class PositronBootstrapExtensionsInitializer extends Disposable {
 		return URI.file(this.environmentService.bootstrapExtensionsPath);
 	}
 
+	/**
+	 * Manages the cleanup of extensions that were bootstrapped in previous release of Positron,
+	 * but have since been removed or replaced.
+	 * Typically, we should only log warnings since the extension ultimately falls under the user's
+	 * control. However, there may be cases when the consequences of having a previously bootstrapped
+	 * extension installed are severe enough to warrant us forcing an uninstall.
+	 */
+	private cleanupOldExtensions(lastKnownVersion: string): void {
+		// If lastKnownVersion is < 2025.06, check for the Black Formatter extension.
```

Comment:
@juliasilge previously I was checking for < 2025.06 even though this was going to be included in 2025.05 because there were many 2025.05 builds that didn't include Ruff and users who installed those wouldn't get Ruff until 2025.06. Now this is going to be in the first 2025.06 build so we don't have to worry about that so I just removed the additional comment rather than modifying the version check as well.